resolwe_bio/processes/clustering/hierarchical_clustering.yml

```diff
@@ -56,3 +56,65 @@
     bash: |
       genehcluster.py {% for e in etcs %}"{{e.etcfile.file}}" {% endfor %}-e {% for e in etcs %}"{{e|id}}" {% endfor %}{% if genes %}-g {{genes|join:" "}}{% endif %} -l {{linkage}} -d {{distance}}
       echo "{\"proc.progress\":1.0,\"proc.rc\":$?}"
+
+
+- slug: clustering-hierarchical-samples
+  name: Hierarchical clustering
```

Comment:
Rename to "Hierarchical clustering of samples". Also rename the slug, description and type of `clustering-hierarchical-bcm` to `clustering-hierarchical-genes-etc`, "Hierarchical clustering of genes" and `data:clustering:hierarchical:genes:etc`, respectfully.